readthedocs/proxito/views/mixins.py

```diff
@@ -128,10 +128,16 @@ def get_redirect(self, project, lang_slug, version_slug, filename, full_path):
         :returns: the path to redirect the request and its status code
         :rtype: tuple
         """
+
+        # Generate the full path from the filename.
+        path = filename
+        if not filename.startswith('/'):
+            path = '/' + filename
```

Comment:
not sure about this here. If the path of the filename is `path/to/filename`, the filename will be `filename` and this will be `/filename`, which isn't the real path.

Also, probably is better to pass the path to `get_redirect` instead of guessing?

Reply:
Yes. This need more thinking. I just wanted to share where was the problem with the current redirects in production when using El Proxito.